backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/coverage/CoverageRegionCollector.kt

```diff
@@ -71,32 +71,57 @@ private class IrFunctionRegionsCollector(
         val irFile: IrFile
 ) : IrElementVisitorVoid {
 
+    val regions = mutableMapOf<IrElement, Region>()
+
     private val irFileStack = mutableListOf(irFile)
 
+    private val regionStack = mutableListOf<Region>()
+
+    private val irStatementsStack = mutableListOf<Pair<IrStatement, IrStatement>>()
```

Comment:
Let's replace `Pair<IrStatement, IrStatement>` with `data class` that has meaningful names for its elements.